api/src/main/resources/marquez/db/migration/V56__alter_job_versions_to_drop_job_context_uuid.sql

```diff
@@ -0,0 +1,3 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+ALTER TABLE job_versions DROP COLUMN job_context_uuid;
```

Comment:
Let's drop column in the next version - this would prevent rollback if the feature would be buggy.